src/js/elements/translations.js

```diff
@@ -0,0 +1,30 @@
+import {Observable} from "../utils";
+import i18next from 'i18next';
+
+export class Translations extends Observable {
+    constructor(translations) {
+        super();
+
+        this.translations = translations;
```

Comment:
can we make this more easy on the user.
currently I would need to define the following:
```json
"translations": {
    "en": {
      "translation": {
        "Point Mapper": "Services"
      }
    }
  },
```
can we turn this into something more like this: (removing the translation key)

```json
"translations": {
    "en": {
      "Point Mapper": "Services"
    }
  },
```

you would then need to change this line here to something like this:
```js
this.translations = {};
Object.keys(translations).forEach((key) => {
  this.translations[key] = { translations: this.translations[key] }
})
```
or a better solution. this is just the fastest I came up with :)